src/main/java/uwu/Ui.java

```diff
@@ -0,0 +1,123 @@
+package uwu;
+
+import uwu.task.Task;
+import uwu.task.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * Text UI of the application.
+ */
+public class Ui {
+    /** The Scanner to read the user's input. */
+    Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Reads the user's input.
+     *
+     * @return The user's input.
+     */
+    public String readCommand() {
+        String userCommand = scanner.nextLine().toLowerCase().trim();
+        return userCommand;
+    }
+
+    /**
+     * Prints a horizontal line to separate chat messages.
+     */
+    public void showLine() {
+        String horizontalLine ="----------------------------------------------------------------------";
+        System.out.println(horizontalLine);
+    }
+
+    /**
+     * Greets users.
+     */
+    public void greetUsers() {
+        String greetings = "\thellu!\n\ti am oo woo <:\n\thow can i be of service today?";
+        System.out.println(greetings);
+    }
+
+    /**
+     * Leaves chat.
```

Comment:
Noticed that for the Java Docs of this function in Ui.java are somewhat ambiguous. Does this function end the program? or does it output a leaving message?